### PR TITLE
Codestyle cleanup 5

### PR DESCRIPTION
**Pull request by noescape00**
`.this` qualifiers and redundant namespaces when calling a method (DoSmth() instead of Lib1.SmthUtils.DoSmth()) for NBitcoin

### CODE REVIEW

File: src/NBitcoin.Tests/ChainTests.cs [hunk 1]
```diff
@@ -187,7 +187,7 @@ public void CanFindFork()
 
         private void AssertFork(ConcurrentChain chain, ConcurrentChain chain2, ChainedHeader expectedFork)
         {
-            ChainedHeader fork = this.FindFork(chain, chain2);
+            ChainedHeader fork = FindFork(chain, chain2);
```
Comment (bokobza) [inline src/NBitcoin.Tests/ChainTests.cs]: hmm I wonder why this. has been removed from here and other places.

Comment (noescape00) [reply src/NBitcoin.Tests/ChainTests.cs]: Hm, that's really odd. 

I've saw resharper chocking on a file a two but executing the rule vise versa is something I didn't see. 

I've fixed  those places manually.

**Comment by bokobza** [comment]
I just restarted the Mac on Travis build, the testSigHashes test fails sometimes.

**Comment by noescape00** [comment]
> the testSigHashes test fails sometimes.

Yes, it is failing when ran with other tests for a month or so.

**Comment by noescape00** [comment]
I reran the tests on my local machine- all passes. 
Let's also wait for the CI just in case.

**Pull request merged**